assets/js/app.js

```diff
@@ -2,396 +2,18 @@
 //import "../css/app.scss"
 
 import "phoenix_html"
-import {
-  Socket
-} from "phoenix"
+import { Socket } from "phoenix"
 import topbar from "topbar"
-import {
-  LiveSocket
-} from "phoenix_live_view"
-
-
+import { LiveSocket } from "phoenix_live_view"
 import L from "leaflet"
 import "leaflet-makimarkers"
 import "@ryangjchandler/alpine-clipboard"
 import "alpinejs"
```

Comment:
Should these imports be in the relevant files?

Reply:
Yes, I should have removed these from app.js.

Reply:
Looks like alpine and alpine clipboard should be left, as it seems to register it globally when imported like this and not like [described here](https://github.com/ryangjchandler/alpine-clipboard#npm).